src/main/java/net/dv8tion/jda/api/Permission.java

```diff
@@ -45,6 +45,7 @@
     BAN_MEMBERS(          2, true, false, "Ban Members"),
     NICKNAME_CHANGE(     26, true, false, "Change Nickname"),
     NICKNAME_MANAGE(     27, true, false, "Manage Nicknames"),
+    TIMEOUT_MEMBERS(     40, true, false, "Timeout Members"),
```

Comment:
Isn't this called `MODERATE_MEMBERS`?

Reply:
We discussed that earlier, it is internally and in the API called moderate_members, but in the client called timeout_members, I guess that can be reverted back to what it was earlier, just keeping the "Timeout Members"

Reply:
I suggest we keep the constant in sync with what they are called in the API, just like all the other permissions

Reply:
We should use what the docs use, even if I dislike this naming.